### PR TITLE
Alert-Avatar

### PR DESCRIPTION
**Pull request by aman11s**
Added two components.
- Alert
i. Simple Alert
ii. Outline Alert

- Avatar
i. Image Avatar
ii. Text Avatar

- Netlify preview link is available below.

### CODE REVIEW

**Review by shaik-saad — APPROVED** [review]
Very clean. Great work Aman!

File: CSS/main.css
Comment (ruchiragoswami) [inline CSS/main.css:87]: I personally avoid using keywords as the names for classes or IDs. For example, main and footer are elements/tags and here they are class-names also. 
What I do is add another word with the tag-name, for example, main-container or anything, so that it is distinctive.

**Review by ruchiragoswami — COMMENTED** [review]
Great work Aman! 
I shared my thought as a comment. With new rules in effect on PR reviews, I will have to check before approval :)

**Review by Kevin-Solomon — APPROVED** [review]

**Review by ruchiragoswami — COMMENTED** [review]
Looks good!

**Review by ruchiragoswami — COMMENTED** [review]
Looks good

**Pull request merged**
